src/server/debug.rs

```diff
@@ -264,7 +264,9 @@ impl Debugger {
 
     /// Scan MVCC Infos for given range `[start, end)`.
     pub fn scan_mvcc(&self, start: &[u8], end: &[u8], limit: u64) -> Result<MvccInfoIterator> {
-        if !start.starts_with(b"z") || (!end.is_empty() && !end.starts_with(b"z")) {
+        if !start.starts_with(b"z")
+            || (!end.is_empty() && !(end.ge(keys::DATA_MIN_KEY) && end.le(keys::DATA_MAX_KEY)))
```

Comment:
Does it same with`!end.starts_with(b"z"))`?

Reply:
No, it does not.
While the original code only accepts ```end``` that starts with ```b"z"```,
the modified one additionally accepts ```end``` that has ```key::DATA_MAX_KEY``` which is ```b"{"``` and does not start with ```b"z"```.
This modification makes ```scan_mvcc()``` be able to scan all possible ```DATA_KEY```.

Reply:
I did not consider that there may exist the region that does not start or end with a table row.
It should be considered.
Thank you for your comment.

I will complement it as follows:
- If  the given region does not contain any ```DATA_KEY```, print a message and do nothing.
- If the given region contains any ```DATA_KEY```, print the MVCC info for those ```DATA_KEY```.

BTW, do we need to support for scanning MVCC keys of non-```DATA_KEY```?

Reply:
> @AndreMouche
> I think I misunderstood your words.
> What is the region which does not start or end with a table row?
> Is the region whose start/end key does not start with `b"z"`?
 yes @jaltabike

Reply:
> > I think I misunderstood your words.
> > What is the region which does not start or end with a table row?
> > Is the region whose start/end key does not start with `b"z"`?
> 
> yes @jaltabike, a region may not start with a table row while ending with a table row.

In the existing code of ```tikv-ctl```,
other functions such as ```compact_region()``` and ```diff_region()``` get the start/end keys of a region from ```RegionInfo``` stored in ```raft-cf```,
and when getting the start/end keys, they seem to assume that both keys start with ```b"z"```.
https://github.com/tikv/tikv/blob/becec689146467090445502f02863d163d30bf54/src/bin/tikv-ctl.rs#L339-L340
So, If these existing functions are correct,
I think all the region may start and end with ```b"z"```.
Could regions that do not start/end with ```b"z"``` exist?

Reply:
I'm not sure.. @zhangjinpeng1987 Can you answer it?

Reply:
@hicqu PTAL
The code I mentioned above seems to be written by you.
Can you answer my question?: "Could regions that do not start/end with b"z" exist?"